cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -21,10 +21,9 @@
 
 [GovernanceSystemSCConfig]
 	ProposalCost = "5000000000000000000" #5 eGLD
-	NumNodes     = 500
-	MinQuorum    = 400
-	MinPassThreshold = 300
-	MinVetoThreshold = 50
+	MinQuorum    = "158000000000000000000" # sqrt(10*NodePrice)
+	MinPassThreshold = "80000000000000000000"
```

Comment:
It's vote power (changed to reflect the comment above). But it's open to be adjusted, in terms of values, did not gave them too much thought.

Reply:
We should put there some real values (now or during the system testing, otherwise we will end up with these values same as we did with the delegation contract initial values)

Reply:
I agree